DA-131: Add a next and prev button in the annotation tool

JeannedArk opened the pull request:
The header in the annotation tool was improved by coloring the project and
document title and adding a next and previous button to navigate quickly
through the documents.

NicklasLinz commented:
The 'completed' checkbox has 5px margin-top. I'll merge it once that is fixed.

NicklasLinz commented:
Also adding a warning that it may take a while (similar to Project page) would be great

JeannedArk committed:
DA-131: Add a next and prev button in the annotation tool

The header in the annotation tool was improved by coloring the project and
document title and adding a next and previous button to navigate quickly
through the documents.
